### PR TITLE
Code files,badges all are present in separate folders

### LINKED ISSUE #37: Uploads folder contain code files and created badges folders
**I'm submitting a ...**  (check one with "x")
- [x ] feature request
**Current behavior:**
 Uploads folder contains code files and created badges folder, which should not happen .

**Expected behavior:**
Uploads should only contain uploaded files and code files should be present in app/

### PR DESCRIPTION
**Pull request by nanspro**
Fixes #37 

#### Short description of what this resolves:
-Code is separated from static files.
-Proper directories arrangement

#### Changes proposed in this pull request:
-Uploads folder now only contains uploaded csv and png files.
-static/badges contain badges formed by uploaded files and also contains all-badges.pdf
-generate-badges.py and merge_badges.sh are now present in app/.
workspace 1_010

### CODE REVIEW

**Comment by nanspro** [comment]
@jajodiaraghav

**Comment by nanspro** [comment]
@jajodiaraghav  Can you review it and tell me any issue if present.

**Comment by jajodiaraghav** [comment]
@nanspro I need some time to review this. By the way, did you test the functionalities of the app after making all the changes?

Please give me some time.

**Comment by nanspro** [comment]
Okay and yeah I check the working of app it works fine

**Comment by nanspro** [comment]
@jajodiaraghav  these codacy issues were also present before i changed the code so do i need to change them to get my PR merged

**Comment by niccokunzmann** [comment]
@nanspro Do you need help? I think this bot is confusing sometimes.

**Comment by nanspro** [comment]
@niccokunzmann  i am stuck in this codacy stuff

File: app/main.py [hunk 1]
```diff
@@ -5,22 +5,29 @@
...
 
+
 @app.route('/')
 def index():
-	return render_template('index.html')
+    return render_template('index.html')
+
 
 def generate_badges():
+<<<<<<< HEAD
```
Comment (niccokunzmann) [inline app/main.py]: This might be an issue, also in the following lines.

**Comment by niccokunzmann** [comment]
Can you create another pull-request and we have a discussion there and ask for help... I really would like to merge this..

**Comment by niccokunzmann** [comment]
What do you mean by codacy? Is this a code quality metric? Where can I find this?

**Pull request closed without merging**